modules/ROOT/pages/to-configure-api-settings.adoc

```diff
@@ -16,6 +16,19 @@ In a private Exchange instance, asset administrators have access to change the v
 
 REST APIs provide a mocking service with data to test API features. The mocking service is always public.
 
+== Configure Non-Managed Instances 
+For published assets not managed by API Manager, asset administrators can add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL and REST APIs. The name, URL, and visibility of a non-managed instance can be changed.
```

Comment:
Can we fix the wording here? API Manager does not manage assets, it manages instances.

Reply:
maybe just a small addition:
```
For published assets with instances not managed by API Manager, ...
```
but I'm not sure if that reads well

Reply:
Yes, this is good. Fixed